spec/integration/_dev/build/build.spec.yml

```
@@ -20,7 +20,17 @@ spec:
               type: string
               description: Source reference
               pattern: '^git@.+'
+            import_common_dynamic_mappings:
```

Nit. `common` looks redundant here.
```suggestion
            import_dynamic_mappings:
```

---

And maybe `dynamic` too, this feature imports mappings for this schema, they happen to be mostly dynamic, but could be static too.
```suggestion
            import_mappings:
```
Or:
```suggestion
            import_template:
```